product_docs/docs/pgd/5/security/role-management.mdx

```diff
@@ -2,36 +2,36 @@
 
-It is especially important that if you do create a role or user in a non-PGD, un-replicated database, that you do not go on to make an object in the PGD replicated database rely on that role. It will break the replication process as PGD cannot replicate a role that is not in the PGD replicated database. 
+If you do create a role or user in a non-PGD, unreplicated database, it's especially important that you don't make an object in the PGD-replicated database rely on that role. It will break the replication process, as PGD can't replicate a role that isn't in the PGD-replicated database. 
 
-This automatic replication behavior can be disabled by turning off the[ bdr.role_replication](https://www.enterprisedb.com/docs/pgd/latest/reference/pgd-settings/#bdrrole_replication) setting but is not recommended.
+You can disable this automatic replication behavior by turning off the [`bdr.role_replication`](https://www.enterprisedb.com/docs/pgd/latest/reference/pgd-settings/#bdrrole_replication) setting. However, we don't recommend that.
 
 
-## **Roles for new nodes**
+## Roles for new nodes
 
-New PGD nodes that are added using[ bdr_init_physical](https://www.enterprisedb.com/docs/pgd/latest/reference/nodes/#bdr_init_physical) will automatically replicate the roles from other nodes of the PGD cluster.
+New PGD nodes that are added using [`bdr_init_physical`](https://www.enterprisedb.com/docs/pgd/latest/reference/nodes/#bdr_init_physical) replicate the roles from other nodes of the PGD cluster.
 
-If a PGD node is joined to a PGD group manually, without using bdr_init_physical, existing roles aren't copied to the newly joined node. This is intentional behavior to ensure that access isn't accidentally granted.
+If a PGD node is joined to a PGD group manually, without using `bdr_init_physical`, existing roles aren't copied to the newly joined node. This is intentional behavior to ensure that access isn't accidentally granted.
 
-Roles, by default, have access to all databases. Automatically copying users to a newly created node would therefore open the possibility of unintentionally exposing a database that exists on the node. If the PGD cluster had a user Alice, and Alice had been automatically created on a newly joined node, by directly connecting to that node, Alice would have access to all the databases on that node. PGD therefore doesn’t copy over existing roles and, in this situation, recommends that you copy over roles manually.
+Roles, by default, have access to all databases. Automatically copying users to a newly created node therefore opens the possibility of unintentionally exposing a database that exists on the node. If the PGD cluster had a user Alice, and Alice was automatically created on a newly joined node, by directly connecting to that node, Alice would have access to all the databases on that node. PGD therefore doesn't copy over existing roles. In this situation, we recommend that you copy over roles manually.
```

Comment:
The would was there because this is something we *don't* do. The sentence now suggests it is possible.

Reply:
```suggestion
Roles, by default, have access to all databases. Automatically copying users to a newly created node would therefore open the possibility of unintentionally exposing a database that exists on the node. If the PGD cluster had a user Alice, and Alice was automatically created on a newly joined node, by directly connecting to that node, Alice would have access to all the databases on that node. PGD therefore doesn't copy over existing roles. In this situation, we recommend that you copy over roles manually.
```